### PR TITLE
[Screen Orientation] Fix: add requestFullscreen to onchange-event-subframe.html

### PR DESCRIPTION
**Pull request by Johanna-hub**

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]

File: screen-orientation/onchange-event-subframe.html [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 <script>
 promise_test(async t => {
+  await test_driver.bless("request full screen", () => {
```
Comment (foolip) [inline screen-orientation/onchange-event-subframe.html]:
```suggestion
  await test_driver.bless("request fullscreen", () => {
```

**Comment by foolip** [comment]
@marcoscaceres did you forget to merge?

**Comment by marcoscaceres** [comment]
Indeed. I can take a look tomorrow at the merge conflicts.

**Comment by marcoscaceres** [comment]
Oh wait, the UI lied to me. No merge conflicts:)

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Mobile view says it’s ok to merge while desktop view says merge conflicts. Anyway, will fix tomorrow.

File: screen-orientation/onchange-event-subframe.html (hunk 1 above)
Comment (foolip) [reply screen-orientation/onchange-event-subframe.html]: @marcoscaceres perhaps this suggestion is why it looked like there were conflicts. I'm not able to apply them and you probably can't either. If so just disregard it.

**Comment by marcoscaceres** [comment]
Ok, recreated it https://github.com/web-platform-tests/wpt/pull/20146 ... merging that one will close this one.

**Pull request closed without merging**